packages/coil-extension/src/background/di/configureContainer.ts

```diff
@@ -38,7 +38,7 @@ export function configureContainer({
   }
 
   container.bind(tokens.CoilDomain).toConstantValue(coilDomain)
-  container.bind(tokens.UserAgent).toConstantValue(navigator.userAgent)
+  container.bind(tokens.UserAgent).toConstantValue('navigator.userAgent')
```

Comment:
WorkerNavigator should be available ??

Reply:
So the problem here was that `Navigator` when used as an injection symbol was no longer available in the worker, as that class is not defined.